tasks-by-users/mmusk/Creative Card/style.css

```diff
@@ -0,0 +1,266 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+
+  font-family: 'Poppins', sans-serif;
+}
+
+body{
+    display: flex;
+    justify-content: center;
+    align-content: center;
+
+    min-height: 100vh;
+
+    background-color: #131313;
+}
```

Comment:
wrap this in one big container its not a good practice to give body properties @mmusk

Reply:
fixed @coder-mezab